src/Detector.php

```diff
@@ -64,6 +64,47 @@ public function determineMimeType(\SplFileObject $file): string
         return 'text/plain; charset=utf-8';
     }
 
+    /**
+     * Read width and height from a image file.
+     *
+     * @param \SplFileObject $file File object
+     * @return array [$width, $height]
+     * @throws \InvalidArgumentException Unreadable SplFileInfo
+     * @throws \RuntimeException SplFileObject $file object is failed to rewind
```

Comment:
I don't see where this exception can be thrown? Can you tell me which line(s) this exception can come from? If it's an old comment, please remove it.

Reply:
https://github.com/ancarda/file-detector/blob/7f1a2a2dd504737e5a11a223ae44b3c5ddcefd2c/src/Detector.php#L87

See PHP documentation for [`SplFileObject::rewind`](https://secure.php.net/manual/en/splfileobject.rewind.php). Originally mentioned in [a different review comment](https://github.com/ancarda/file-detector/pull/8#discussion_r214002667).